.github/workflows/test-install.yml

```diff
@@ -38,29 +38,3 @@ jobs:
         run: |
           cp .github/docker-compose.dry-run.yml docker-compose.override.yml
           docker compose up --attach ac-db-import --attach ac-worldserver --attach ac-authserver
-
-  test-dev-install-process:
-    env:
-      COMPOSE_DOCKER_CLI_BUILD: 1
-      DOCKER_BUILDKIT: 1
-      BUILDKIT_INLINE_CACHE: 1
-    runs-on: ${{ matrix.os }}
-    strategy:
-      matrix:
-        include:
-          # - os: macos-latest TODO: install docker
-          # - os: ubuntu-20.04
-          - os: ubuntu-latest
-          # - os: windows-latest TODO: install docker
-    steps:
-      - uses: actions/checkout@v2
-
-      # TODO: fix after this is solved: https://github.com/docker/compose/issues/9515#issuecomment-1303538917
-      - name: Test
-        shell: bash
-        run: |
-          cp .github/docker-compose.dev-server.yml docker-compose.override.yml
-          cp .github/config.sh config.sh
-          docker compose up -d ac-dev-database
-          docker compose run --no-deps ac-dev-server bash -c "./acore.sh compiler build"
-          docker compose run --no-deps ac-dev-server bash -c "./env/dist/bin/dbimport"
```

Comment:
why did you remove this? for incompatibility reasons?

Reply:
yes. If the dev resources are removed from the repo, it means that tests for them will fail